### PR TITLE
feat(curriculum): add CSS Flexbox Quiz

### LINKED ISSUE #56187: [HACKTOBERFEST 2024] - CSS Flexbox Quiz
> [!IMPORTANT]
> This issue is part of our 2024 Hacktoberfest initiative. If you are interested in working on this issue, refer to our [top-level tasks](https://github.com/freeCodeCamp/freeCodeCamp/issues/56170) to see if this issue is claimed, or to claim it if it is not.

This issue is to create the CSS Flexbox quiz.

The file you should be working on is found in `/curriculum/challenges/english/25-front-end-development/quiz-css-flexbox`.

**All quiz questions need to relate to one of these topics:**

- what is flexbox
- common flex properties
  - flex-wrap
  - justify-content
  - align-items

> [!CAUTION]
> Questions which do not fall under these topics will be rejected.

You are welcome to include code blocks in your questions, but please keep them short (four lines or less).

Please do NOT open a pull request until you have completed all questions. Your quiz should have **exactly 20 questions**.

### PR DESCRIPTION
**Pull request by COSMIC-ATOM97**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #56187

### CODE REVIEW

**Comment by ilenia-magoni** [comment]
Please use backticks around anything code

**Comment by COSMIC-ATOM97** [comment]
I’ve made the required updates to the file. I would appreciate it if you could take a moment to review it.

File: curriculum/challenges/english/25-front-end-development/quiz-css-flexbox/66ed8fe7f45ce3ece4053eb2.md [hunk 1]
```diff
@@ -17,439 +17,438 @@ Answer all of the questions below correctly to pass the quiz.
...
 ---
 
-Placeholder distractor 3
+`flex-wrap`
 
 #### --answer--
 
-Placeholder answer
-
+`align-items`
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-css-flexbox/66ed8fe7f45ce3ece4053eb2.md:454]:
```suggestion
`align-items`

```
We need the EOF new line.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
One tiny nit, then this should be good to go!

**Review by naomi-lgbt — APPROVED** [review]

File: curriculum/challenges/english/25-front-end-development/quiz-css-flexbox/66ed8fe7f45ce3ece4053eb2.md [hunk 2]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
...
 
 ---
 
-Placeholder distractor 3
+`flex-start`
 
 #### --answer--
 
-Placeholder answer
+`space-evenly`
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/quiz-css-flexbox/66ed8fe7f45ce3ece4053eb2.md:432]: it is also an answer on line 80, this question is a duplicate and needs to be changed to something else

File: curriculum/challenges/english/25-front-end-development/quiz-css-flexbox/66ed8fe7f45ce3ece4053eb2.md [hunk 3]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
...
 
 ---
 
-Placeholder distractor 3
+`flex-end`
 
 #### --answer--
 
-Placeholder answer
+`nowrap`
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/quiz-css-flexbox/66ed8fe7f45ce3ece4053eb2.md:410]: duplicate question, it is an answer on line 36

**Comment by gikf** [comment]
Hey @COSMIC-ATOM97,

did you have a chance too look at the latest feedback?

**Comment by ilenia-magoni** [comment]
@COSMIC-ATOM97 We would have really appreciated if you could see the PR to completion, deleting the repo closes the PR

**Pull request closed without merging**